packages/rancher-monitoring/66.7/rancher-monitoring/package.yaml

```diff
@@ -1,7 +1,7 @@
 url: https://github.com/prometheus-community/helm-charts.git
 subdirectory: charts/kube-prometheus-stack
 commit: 9c2619e6f3650a5722b0f81f18850b3751ce31ba
-version: 66.7.1-rancher.2
+version: 66.7.1-rancher.3
```

Comment:
Do we need to increment the version if `rancher.2` is not yet on rancher/charts?

Reply:
Oh good catch.